lib/src/local_working_copy.rs

```diff
@@ -1223,6 +1223,7 @@ impl TreeState {
         &self,
         disk_path: &Path,
         conflict_data: Vec<u8>,
+        executable: bool,
```

Comment:
copy to `FileState::for_file(_, ..)`?

Reply:
right, i missed the hardcoded `false` there